### PR TITLE
Replace MaltegoOAuth's pycrptodome dependency with cryptography lib

### PR DESCRIPTION
**Pull request by tendai-zw**
Removed the pycrptodome dependency for MaltegoOAuth which has large dependencies on Windows OS. I replaced it with cryptography https://cryptography.io/en/latest/

### CODE REVIEW

File: maltego_trx/oauth.py [hunk 1]
```diff
@@ -25,24 +26,29 @@ def _rsa_decrypt(private_key_path=None, ciphertext=None):
...
-        unpad = lambda s: s[:-ord(s[len(s) - 1:])]
         key = base64.b64decode(key)
+        cipher = Cipher(algorithms.AES(key), modes.ECB(), backend=default_backend())
+        decryptor = cipher.decryptor();
         ciphertext = base64.b64decode(ciphertext)
-        cipher = AES.new(key, AES.MODE_ECB)
-        plaintext = unpad(cipher.decrypt(ciphertext)).decode('utf8')
+        padded_b64_plaintext = decryptor.update(ciphertext) + decryptor.finalize()
+
+        unpad = lambda data: data[:-ord(data[len(data) - 1:])]
```
Comment (totakura) [inline maltego_trx/oauth.py:50]: This looks like PKCS7 padding. Why not use `cryptography.hazmat.primitives.padding.PKCS7()`?

**Review by totakura — CHANGES_REQUESTED** [review]
@tendai-zw please check.

File: maltego_trx/oauth.py (hunk 1 above)
Comment (tendai-zw) [reply maltego_trx/oauth.py]: Ack

**Comment by tendai-zw** [comment]
@totakura change requests resolved.

**Review by totakura — APPROVED** [review]
LGTM.

**Pull request merged**